frontend/src/assets/css/colors.scss

```diff
@@ -0,0 +1,11 @@
+:root {
+  --general-dark: #1d1d1d;
```

Comment:
replace it with Sass Variables

Reply:
I did that, but now when I import variables via `@import "vars";` i get error from eslint:
Expected ""vars.scss"" to be "url("vars.scss")"
But when I import via `@import url("./vars.scss");` I get an error:
[plugin:vite:css] [postcss] [sass] Undefined variable.

Reply:
I also tried using the `@use` syntax but stylelint throws an error "Unexpected unknown at-rule "@use" at-rule-no-unknown".